### ISSUE TITLE
Could you add this to the Edge Brower Store?

### ISSUE THREAD

**Issue by lizhendong128** [nsfw-filter/nsfw-filter#183]
I like this, but I usually use the Edge brower. Could you add this to the Edge Brower Store~
Very thanks~~

**Comment by omeritzics** [comment]
Joining the request, this will be great.

Until then, you can install NSFW Filter for Edge from the Google Chrome Web Store.

**Comment by navendu-pottekkat** [comment]
Thank you for raising this issue @lizhendong128 @omeritzics 

This seems like something we can do easily. I will look into publishing it to Edge. For now, I think you can install the Chrome extension on your Edge browser. See https://pureinfotech.com/install-chrome-extension-chromium-edge/

**Comment by Mahdi-Noor-Hasan** [comment]
> I like this, but I usually use the Edge brower. Could you add this to the Edge Brower Store~
> Very thanks~~

Why do we need it? Can't we just install it from Chrome Web Store?

**Comment by navendu-pottekkat** [comment]
Yes we can, but it would be better if we have it listed on the default Edge store as well if that is a thing.

**Comment by Delphox** [comment]
Another good reason to upload to Edge Store is an alternative in case the Chrome Webstore version is down (like right now). They follow slightly different rules.

**Comment by YegorZaremba** [comment]
@navendu-pottekkat Is it time to do it?

**Comment by navendu-pottekkat** [comment]
I will look into this
